managing-os/templates/archlinux.md

```diff
@@ -16,27 +16,666 @@ be considered experimental as Qubes developers team use mainly Fedora-based VMs
+            *   Deselect '**mgnt_salt**'
+            
+            *   Select '**archlinux**'
+
+            *   Select '**OK**' Press **Enter**
+<br>
+<br>
+![arch-template-13](/attachment/wiki/ArchlinuxTemplate/arch-template-13.png)
+<br>
+<br>
+
+        *   Screen '**Get Resources**' wants to download additional packages needed for the choosen plugin/s.
+        
+            *   Select '**OK**' Prss '**Enter**'
+            
+        *   Upon completion you will get choose '**OK**' to proceed to the next screen
+        
+            *   Press '**Enter**'
+<br>
+<br>
+![arch-template-14](/attachment/wiki/ArchlinuxTemplate/arch-template-14.png)
+<br>
+![arch-template-15](/attachment/wiki/ArchlinuxTemplate/arch-template-15.png)
+<br>
+<br>
+
+        *   Screen '**Template Distribution Selection**' allows you to choose the actual template/s you wish to build.
+
+            *   Scroll Down to the very bottom (it is off the screen at first)
+
+            *   Select '**archlinux**'
+
+            *   Select '**OK**' Press '**Enter**'
+<br>
+<br>
+![arch-template-16](/attachment/wiki/ArchlinuxTemplate/arch-template-16.png)
+<br>
+<br>
+
+            *Note: 'Setup' will close and will output the text of the created build.conf file as well as the needed                    **make** commands to build the template*
+<br>
+<br>
+![arch-template-17](/attachment/wiki/ArchlinuxTemplate/arch-template-17.png)
+<br>
+<br>
+<br>
+
+#####**7:   Install all the dependencies:**##### 
+
+*Note: make sure you are in the “qubes-builder” directory to run the following cmds*
+
+*   **$ make install-deps**
+<br>
+<br>
+![arch-template-18](/attachment/wiki/ArchlinuxTemplate/arch-template-18.png)
+<br>
+<br>
+<br>
+
+#####**8:   Get all the require sources for the build: (Note: this may take some time)**#####
+
+*   **$ make get-sources**
+<br>
+<br>
+![arch-template-19](/attachment/wiki/ArchlinuxTemplate/arch-template-19.png)
+<br>
+<br>
+<br>
+
+#####**9:   Make all the require Qubes Components:**#####
+
+*   **Note:** You can run a single command to build all the Qubes components or you can run them each individually.
+     Both ways below:
+
+    *   Single command to build all Qubes components together: (this command can take a long time to process depending of your pc proccessing power)
+    
+        *   **$ make qubes-vm**
+        <br>
+        <br>
+![arch-template-20](/attachment/wiki/ArchlinuxTemplate/arch-template-20.png)
+        <br>
+        <br>
+        
+        
+            *   These are the indivual component 'make' commands:
+    
+                *   **$ make vmm-xen-vm**
+          
+                *   **$ make core-vchan-xen-vm**
+           
+                *   **$ make core-qubesdb-vm**
+           
+                *   **$ make linux-utils-vm**
+           
+                *   **$ make core-agent-linux-vm**
+           
+                *   **$ make gui-common-vm**
+           
+                *   **$ make gui-agent-linux-vm**
```

Comment:
This looks weird (and other inline code). Generally it's better to use `` type quotation for code, instead of bold.